lecture6.tex

```diff
@@ -221,7 +221,6 @@ \section{Graph distances}
 \section{Exercises}
 %TODO
```

Comment:
Should probably remove this comment as well.

As a general best practices thing, keeping to-do lists or notes to self in the actual code is going to come back to bite you. (I speak from experience here - my first paper is full of such notes and it is a *horrible unreadable mess* if you look at the TeX source, and the todos get forgotten or lost, not done. I have *regrets* about how that paper was written... 🥲) 

Try to use a separate system for that - GitHub has a built-in issue tracker that'd probably work pretty okay for a project like this.

Reply:
Thanks for the advice! Should be fixed now.